addons/web_editor/static/src/js/editor/odoo-editor/src/utils/sanitize.js

```diff
@@ -231,7 +231,7 @@ export function sanitize(nodeToSanitize, root = nodeToSanitize) {
         let node = isList ? block.parentElement : block;
 
         // Sanitize the tree.
-        while (node?.isConnected && root.contains(node)) {
+        while (root.contains(node)) {
```

Comment:
I get the need for this for `getValue` but I'm afraid it would be costly when we don't need it. Maybe a good in between would be:

```suggestion
        while (node && !(root.isConnected && !node.isConnected) && root.contains(node)) {
```

So we only care about the node being connected if the root itself is as well.